cmd/eyes/main.go

```diff
@@ -0,0 +1,45 @@
+package main
+
+import (
+	"os"
+
+	"github.com/tendermint/tmlibs/cli"
+
+	"github.com/tendermint/basecoin"
+	"github.com/tendermint/basecoin/cmd/basecoin/commands"
+	"github.com/tendermint/basecoin/modules/base"
+	"github.com/tendermint/basecoin/modules/etc"
+	"github.com/tendermint/basecoin/stack"
+)
+
+// BuildApp constructs the stack we want to use for this app
+func BuildApp() basecoin.Handler {
+	return stack.New(
+		base.Logger{},
+		stack.Recovery{},
+	).
```

Comment:
Yes, to run it as a separate process. Mainly for testing. 

Currently merkleeyes is used for two things. A storage layer for base coin (no more in unstable) and a backend abci app for testing tendermint. 

This would replace the second usage and consolidate all abci apps in the new framework.